src/applications/search/components/SimplePagination.jsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+
+import { PAGE_SIZE } from '../constants';
+
+const SimplePagination = ({ prevOffset, nextOffset, handlePageChange }) => (
+  <div className="simple-pagination">
+    {prevOffset && (
+      <a href="#" rel="prev" onClick={handlePageChange(prevOffset)}>
+        « Previous
+      </a>
+    )}
+    <span className="current-page">
+      Page {prevOffset ? prevOffset / PAGE_SIZE + 1 : 1}
+    </span>
+    {nextOffset && (
+      <a href="#" rel="next" onClick={handlePageChange(nextOffset)}>
+        Next »
+      </a>
+    )}
```

Comment:
Since the page changes, it seems like an `<a>` is the right element, but I'm wondering if that `href` value should be calculated or not. Probably low priority, but wondering if there's an accessibility hit.

Reply:
I debated constructing a proper href here, but given that that results pages are transient and prone to change, used the simpler approach of handling the click and having the router handle the page change instead. Also has a better UX because this doesn't result in a full page reload